app/controllers/planning_applications/validation/document/redactions_controller.rb

```diff
@@ -34,7 +35,17 @@ def planning_application_params
         end
 
         def documents_params
-          %i[file redacted publishable validated]
+          [:file, :redacted, :publishable, :validated, tags: []]
+        end
+
+        def filter_tags!
```

Comment:
I'm a bit confused with what this method is trying to achieve. Are we making this more complicated than necessary?

Reply:
It seems like the problem here is that the tags are passed back through a hidden field, which joins the array into a string, and therefore needs to be split back out.

It's probably worth refactoring this later though.